### ISSUE TITLE
STAT errors when NFS Mounting from OSX High Sierra to Vagrant/Ubuntu Trusty.  apt-cache raises false "cache full" error 

### ISSUE THREAD

**Issue by misterkbar** [hashicorp/vagrant#9722]
NFS mounting from OS-X High Sierra to Ubuntu Trusty on Vagrant (possibly more) causes _stat_ to report empty free space on the mountpoint. Have verified that the mounted filesystem is writable.

This affects APT when the APT cache is configured as an NFS mount (to allow for different Vagrant systems to share the same APT cache). _**apt-cache**_ sees the "empty" free space and dies when trying to install a new package.

This problem **_did not exist_** with OS-X/(Non-high) Sierra mounting on the same Vagrant system.

The current fix is to just disable the APT cache NFS sharing.

Have not yet verified if this is a Trusty issue or a Trusty-on-Vagrant issue, as I have no available setup to NFS mounting on a non-vagrant Trusty machine, nor the time to make one.

(Providing basic info, as this is the quickest way to specify the problem):

- Vagrant version: Vagrant 1.9.1
- OS Version running on Vagrant: 
```
    Distributor ID:	Ubuntu
    Description:	Ubuntu 14.04.5 LTS
    Release:	14.04
    Codename:	trusty
```

Problem: running stat (1) returns incorrect free space: (problem is highlighted by **)

```
# stat -f <NFS mounted directory>
File: <directory>
    ID: 0        Namelen: 255     Type: fuseblk
Block size: 65536      Fundamental block size: 65536
Blocks: Total: 3825159    **Free: 0**          Available: 1529840
Inodes: Total: 9223372036854775807 Free: 9223372036853008903
```

```
# df <directory>
Filesystem             1K-blocks      Used Available Use% Mounted on
<machine>:<remote dir> 244810176 244810176  97903360  72% <directory>
```

**Comment by misterkbar** [comment]
Unfortunately, I have no real opportunity to do this, do the the firewall rules in my org. I SUPPOSE I can track down an associate running ubuntu on h{is,er} laptop and try it out. Will do so and get back to you.

**Comment by briancain** [comment]
@misterkbar - No sorry, I mean if you manually mount the share using NFS on a local vm from your host system, do you still experience this behavior? My guess is this is not a Vagrant bug, but a bug with macOS.

**Comment by briancain** [comment]
Hey there,

I am going to close this due to lack of response. If this is still occurring, please open a new issue and follow the provided issue template that appears when you click the "New Issue" button. This will help us in getting a reproduction and fix. Thanks!
